internal/mode/static/telemetry/collector_test.go

```diff
@@ -451,6 +477,7 @@ var _ = Describe("Collector", Ordered, func() {
 								OwnerReferences: nil,
 							},
 						},
+						kubeNamespace,
```

Comment:
This may be up for discussion with following telemetry tests, but as is you don't need to add this `kubeNamespace` here as when `dataCollector.Collect` gets called, it should error before even running `collectClusterID` so this is never used. Same goes for the following tests which you added kubeNamespace.